pages/linux/ledctl.md

```diff
@@ -0,0 +1,20 @@
+# ledctl
+
+> Intel(R) Enclosure LED Control Application.
+> More information: <https://manned.org/ledctl>.
+
+- Turn on the "Locate" LED for specified device(s):
+
+`sudo ledctl locate={{/dev/sda,/dev/sdb...}}`
```

Comment:
```suggestion
`sudo ledctl locate={{/dev/sda,/dev/sdb,...}}`
```

Reply:
Should this really be done, the user may assume you need to finish with a comma like `sudo ledctl locate=/dev/sda,`

Reply:
@CleanMachine1 We use `,...` in `mongoexport`, `gh secret`, `useradd`, etc.

Reply:
That seems like a bad idea to me, personally I would run the command with a , if I saw that but if its the standard then yes use it, it can always be changed if required